plugins/newsdownloader.koplugin/feed_config.lua

```diff
@@ -0,0 +1,8 @@
+return {
+    -- list your feeds here:
+    -- only supports http URL for now
+    -- Atom is currently not supported, only RSS
+    -- set limit to "0" means no download, "-1" no limit.
+    { "http://www.huffingtonpost.com/feeds/index.xml", limit = 2 },
```

Comment:
Wikipedia says this portal is ["overtly liberal/left commentary"](https://en.wikipedia.org/wiki/The_Huffington_Post). 
I don't want this plugin to be connected with politician option.

Let's use something that's neutral view if not then let's replace that with sport source.